lib/compat/wordpress-5.9/block-template-utils.php

```diff
@@ -323,7 +323,18 @@ function _get_block_templates_files( $template_type ) {
 				);
 
 				if ( 'wp_template_part' === $template_type ) {
-					$template_files[] = _add_block_template_part_area_info( $new_template_item );
+					$default_headers = array(
```

Comment:
Before looking at implementation, I'd like to confirm direction. This is just a way to make this work for testing.